mcstatus/server.py

```diff
@@ -53,12 +53,20 @@ def lookup(cls, address: str, timeout: float = 3) -> Self:
         addr = Address.parse_address(address, default_port=cls.DEFAULT_PORT)
         return cls(addr.host, addr.port, timeout=timeout)
 
+    @classmethod
+    @abstractmethod
+    def kind(cls) -> str: ...
```

Comment:
Why not just `isinstance(server, JavaServer)`?

Reply:
Sure.